src/main/java/com/shootoff/camera/cameratypes/PS3EyeCamera.java

```diff
@@ -0,0 +1,621 @@
+//written by ifly53e
```

Comment:
The file has to have the ShootOFF license.  You can include your name before or after the license stating that you wrote the file originally.